vispy/scene/__init__.py

```diff
@@ -30,8 +30,6 @@
 
 """
 
-__all__ = ['SceneCanvas', 'Node']
```

Comment:
Eventually we will probably want to re-populate this so our docs aren't full of junk from all those `import *` calls below...